src/main/java/org/sonar/plugins/stash/StashPluginUtils.java

```diff
@@ -0,0 +1,34 @@
+package org.sonar.plugins.stash;
+
+import java.text.DecimalFormat;
+import java.text.MessageFormat;
+import java.text.ParseException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class StashPluginUtils {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)